client/assets/styles.css

```diff
@@ -967,3 +968,68 @@ body {
     opacity: 0;
     transition: opacity 1s 2s;
 }
+
+.icon {
+    background: white no-repeat center;
+    width: 16px;
+    height: 16px;
+    display: block;
+    margin: 6px auto;
+    opacity: 0.6;
+    transition: 0.1s transform;
+}
+
+.icon:hover {
+    transform: scale(1.1);
+}
+
+[disabled] > .icon,
+.icon.icon--disabled {
+    transform: none;
+}
+
+/*
+SVG icons was downloaded from https://fontawesome.com/icons
+TODO You must give appropriate credit,
```

Comment:
What is considered 'appropriate credit'?

Reply:
Oh, actually nothing. https://fontawesome.com/license/free


> Attribution
> Attribution is required by MIT, SIL OFL, and CC BY licenses. Downloaded Font Awesome Free files already contain embedded comments with sufficient attribution, so **you shouldn't need to do anything additional** when using these files normally.
> 
> We've kept attribution comments terse, so we ask that you do not actively work to remove them from files, especially code. They're a great way for folks to learn about Font Awesome.
>

Reply:
I will remove TODO then